### PR TITLE
iOS release 1.3.0

### PR DESCRIPTION
**Pull request by pavel-kroh-smartlook**
Yesterday, I've silently released new iOS version.

Main topics:
- reset session and user
- add event tracking modes
- add no_rendering rendering mode
- fix memory issues reported by devexperts

Changelog: https://github.com/smartlook/smartlook-ios-sdk

### CODE REVIEW

File: _docs/sdk/ios.md [hunk 1]
```diff
@@ -131,6 +133,36 @@ It is harmless to call the methods in unruly order (e.g., to _start_ while alrea
 
 There is no need to manually stop recording when the app gets suspended by the user by pressing Home button or by switching to another app, or restart recording when the app wakes up. This happens automatically.
 
+### Recording session and user lifecycle
+
+User activities are recorded in sessions. Typical session starts with the app launch, and ends after user leaves the app (thchnically, the app gets in background). 
+
+However, to cover a scenario when user activity in the app is interrupted for a minute by incomming call or message, Smartlook attempts to continue in the session even after the app re/launches after some short time in background.
+
+Also, user is tied to the session. In other words, each session has only one user. Changing user id in the middle of session replaces the previous user of the whole session. New are sessions implicitelly associated with the previous session user.
```
Comment (Acro) [inline _docs/sdk/ios.md:142]: Should be `implicitly`

File: _docs/sdk/ios.md [hunk 2]
```diff
@@ -131,6 +133,36 @@ It is harmless to call the methods in unruly order (e.g., to _start_ while alrea
...
+
+For some apps, this continual sessions are not desired, or they need to change user in middle of the app lifecycle (e.g., on shared tablet in factory where various users log in/out to the running app). Here is a set of method how to control the session and user lifecycle explicitelly from the app.
```
Comment (Acro) [inline _docs/sdk/ios.md:144]: Should be `explicitly`

File: _docs/sdk/ios.md (hunk 1 above)
Comment (pavel-kroh-smartlook) [reply _docs/sdk/ios.md]: Corrected.

File: _docs/sdk/ios.md (hunk 2 above)
Comment (pavel-kroh-smartlook) [reply _docs/sdk/ios.md]: Corrected.

**Pull request merged**